### PR TITLE
Updated Readme with installation instructions

### PR DESCRIPTION
**Pull request by johnflorin**
This is what worked for me yesterday, let me know if I've missed something. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+Installation instructions:
+
+1. Install HACS for Home Assistant
+2. Go to Community-Store-Grocy
+3. Install Grocy
+4. Restart Home Assistant
+5. Go to Grocy-Wrench icon-Manage API keys-Add
+6. Copy resulting API key and Grocy URL and input this in configuration.yaml:
+
 grocy:
```
Comment (SebRut) [inline README.md]: could you readd the formatting for better readability?

**Comment by SebRut** [comment]
Thank you for your contribution :)

File: README.md (hunk 1 above)
Comment (johnflorin) [reply README.md]: Done, hope it's correct now, this is my first time with both the GitHub text editor and the pull request system :)

File: README.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
+<b>Installation instructions:</b>
```
Comment (SebRut) [inline README.md]: Could you change this too a h2 headline using `#`?
e.g. `## Instructions`

**Pull request merged**